.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -52,6 +55,19 @@ fun BuildFeatures.publishBuildStatusToGithub(model: CIBuildModel) {
     }
 }
 
+fun BuildFeatures.triggeredOnPullRequests() {
+    pullRequests {
+        vcsRootExtId = "Gradle_Branches_GradlePersonalBranches"
+        provider = github {
+            authType = token {
+                token = "credentialsJSON:5306bfc7-041e-46e8-8d61-1d49424e7b04"
+            }
+            filterAuthorRole = PullRequests.GitHubRoleFilter.MEMBER
+            filterTargetBranch = masterReleaseBranchFilter
```

Comment:
This looks wrong as this should run the branch of the PR? (Maybe no filter is needed here at all?)

Reply:
It's "target branch filter", but yes, we can of source use "all branch" here. According to [the doc](https://www.jetbrains.com/help/teamcity/branch-filter.html), all branch filter is 

```
+:*
```

Reply:
Fixed, PTAL!